src/modules/invocation/domain/invocation.domain.ts

```diff
@@ -43,4 +43,10 @@ export class Invocation extends Base {
     this.id = id;
     this.selectedMethod = selectedMethod;
   }
+
+  getContractIdValue = (inputString: string): string => {
+    const regex = /{{(.*?)}}/g;
+    const contractId = inputString.replace(regex, (match, text) => text);
+    return contractId;
+  };
 }
```

Comment:
You must to create unit test to this method

Reply:
The test I created in invocation.e2e.spec.ts demonstrates this functionality